features/daily/user_one_call.feature

```diff
@@ -0,0 +1,55 @@
+Feature: User API - user managed with one HTTP call
+
+
```

Comment:
You can follow [STYLEGUIDE](https://github.com/wazo-platform/wazo-acceptance/blob/master/STYLEGUIDE.md)  :)